### ISSUE TITLE
Issue with DO 1.29.0 unable to find /mgmt/tm/sys/provision, rollback fails

### ISSUE THREAD

**Issue by mikeshimkus** [F5Networks/f5-declarative-onboarding#306]
### Environment
 * Declarative Onboarding Version: 1.29.0
 * BIG-IP Version: 16.1.2.1 build 0010
 * Google Cloud instance n1-standard-8

### Summary
When declaring the following config with DO 1.29.0, got an error in the restnoded.log that /mgmt/tm/sys/provision couldn't be found, followed by this: 

Wed, 20 Apr 2022 20:26:34 GMT - severe: [f5-declarative-onboarding: networkHandler.js] Error processing network declaration: tryUntil: max tries reached: tryUntil: max tries reached: \<?xml version="1.0" encoding="ISO-8859-1"?\>
Wed, 20 Apr 2022 20:26:34 GMT - severe: [f5-declarative-onboarding: declarationHandler.js] Error processing declaration: tryUntil: max tries reached: tryUntil: max tries reached: \<?xml version="1.0" encoding="ISO-8859-1"?\>
Wed, 20 Apr 2022 20:26:34 GMT - severe: [f5-declarative-onboarding: restWorker.js] Error onboarding: tryUntil: max tries reached: tryUntil: max tries reached: \<?xml version="1.0" encoding="ISO-8859-1"?\>

I saw this about 6 times in a row with 1.29.0. The same declaration works fine in DO 1.28.0.

Note: A similar config that did not include clustering config seems to work fine. Also, if I wait a couple of minutes and re-submit the configuration, it works.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
    "schemaVersion": "1.0.0",
			"class": "Device",
			"async": true,
			"label": "FAILOVER 3NIC BIG-IP declaration for Declarative Onboarding with PAYG license with example Application",
			"Common": {
				"class": "Tenant",
				"My_DbVariables": {
					"class": "DbVariables",
					"provision.extramb": 1000,
					"restjavad.useextramb": true,
					"config.allow.rfc3927": "enable",
					"dhclient.mgmt": "disable",
					"tm.tcpudptxchecksum": "Software-only"
				},
				"My_Provisioning": {
					"class": "Provision",
					"ltm": "nominal",
					"asm": "nominal"
				},
				"My_Ntp": {
					"class": "NTP",
					"servers": ["0.pool.ntp.org", "1.pool.ntp.org"],
					"timezone": "UTC"
				},
				"My_Dns": {
					"class": "DNS",
					"nameServers": ["169.254.169.254"],
					"search": ["f5.com"]
				},
				"My_System": {
					"autoPhonehome": true,
					"class": "System",
					"hostname": "dd-ca02943f-bigip1.c.f5-7656-pdsoleng-dev.internal",
					"consoleInactivityTimeout": 0,
					"cliInactivityTimeout": 0,
					"autoCheck": true,
					"tmshAuditLog": true,
					"guiAuditLog": false,
					"mcpAuditLog": "enable",
					"preserveOrigDhcpRoutes": false
				},
				"admin": {
					"class": "User",
					"userType": "regular",
					"shell": "bash"
				},
				"externalGwRoute": {
					"class": "Route",
					"target": "external",
					"network": "10.0.1.1/32",
					"mtu": 1460,
					"localOnly": false
				},
				"externalIntRoute": {
					"class": "Route",
					"gw": "10.0.1.1",
					"network": "10.0.1.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"default": {
					"class": "Route",
					"gw": "10.0.1.1",
					"network": "default",
					"mtu": 1460,
					"localOnly": false
				},
				"internalGwRoute": {
					"class": "Route",
					"target": "internal",
					"network": "10.0.2.1/32",
					"mtu": 1460,
					"localOnly": false
				},
				"internalIntRoute": {
					"class": "Route",
					"gw": "10.0.2.1",
					"network": "10.0.2.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"appRoute": {
					"class": "Route",
					"gw": "10.0.2.1",
					"network": "10.0.3.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"external": {
					"class": "VLAN",
					"tag": 4094,
					"mtu": 1460,
					"interfaces": [{
						"name": "1.0",
						"tagged": false
					}],
					"autoLastHop": "default",
					"cmpHash": "default",
					"failsafeEnabled": false,
					"failsafeAction": "failover-restart-tm",
					"failsafeTimeout": 90
				},
				"external-self": {
					"class": "SelfIp",
					"address": "10.0.1.11/32",
					"vlan": "external",
					"allowService": ["tcp:80", "tcp:443", "tcp:4353", "udp:1026"],
					"trafficGroup": "traffic-group-local-only"
				},
				"internal": {
					"class": "VLAN",
					"tag": 4093,
					"mtu": 1460,
					"interfaces": [{
						"name": "1.2",
						"tagged": false
					}],
					"autoLastHop": "default",
					"cmpHash": "default",
					"failsafeEnabled": false,
					"failsafeAction": "failover-restart-tm",
					"failsafeTimeout": 90
				},
				"internal-self": {
					"class": "SelfIp",
					"address": "10.0.2.11/32",
					"vlan": "internal",
					"allowService": ["udp:1026", "tcp:4353"],
					"trafficGroup": "traffic-group-local-only"
				},
				"configSync": {
					"class": "ConfigSync",
					"configsyncIp": "10.0.1.11"
				},
				"failoverAddress": {
					"class": "FailoverUnicast",
					"address": "10.0.1.11",
					"port": 1026
				},
				"failoverGroup": {
					"class": "DeviceGroup",
					"type": "sync-failover",
					"members": ["dd-ca02943f-bigip1.c.f5-7656-pdsoleng-dev.internal", "dd-ca02943f-bigip2.c.f5-7656-pdsoleng-dev.internal"],
					"owner": "/Common/failoverGroup/members/0",
					"autoSync": true,
					"saveOnAutoSync": false,
					"networkFailover": true,
					"fullLoadOnSync": false,
					"asmSync": false
				},
				"trust": {
					"class": "DeviceTrust",
					"localUsername": "admin",
					"remoteHost": "/Common/failoverGroup/members/0",
					"remoteUsername": "admin"
				}
			}
}
```

2. Observe the following error response:
```json
{
	"code": 500,
	"body": {
		"id": "acab6ed3-411e-4860-a5d0-cf91188fb837",
		"selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/acab6ed3-411e-4860-a5d0-cf91188fb837",
		"code": 500,
		"status": "ERROR",
		"message": "rollback failed",
		"errors": "HTML Tag-like Content in the Request URL/Body",
		"result": "HTML Tag-like Content in the Request URL/Body",
		"declaration": {
			"schemaVersion": "1.0.0",
			"class": "Device",
			"async": true,
			"label": "FAILOVER 3NIC BIG-IP declaration for Declarative Onboarding with PAYG license with example Application",
			"Common": {
				"class": "Tenant",
				"My_DbVariables": {
					"class": "DbVariables",
					"provision.extramb": 1000,
					"restjavad.useextramb": true,
					"config.allow.rfc3927": "enable",
					"dhclient.mgmt": "disable",
					"tm.tcpudptxchecksum": "Software-only"
				},
				"My_Provisioning": {
					"class": "Provision",
					"ltm": "nominal",
					"asm": "nominal"
				},
				"My_Ntp": {
					"class": "NTP",
					"servers": ["0.pool.ntp.org", "1.pool.ntp.org"],
					"timezone": "UTC"
				},
				"My_Dns": {
					"class": "DNS",
					"nameServers": ["169.254.169.254"],
					"search": ["f5.com"]
				},
				"My_System": {
					"autoPhonehome": true,
					"class": "System",
					"hostname": "dd-ca02943f-bigip1.c.f5-7656-pdsoleng-dev.internal",
					"consoleInactivityTimeout": 0,
					"cliInactivityTimeout": 0,
					"autoCheck": true,
					"tmshAuditLog": true,
					"guiAuditLog": false,
					"mcpAuditLog": "enable",
					"preserveOrigDhcpRoutes": false
				},
				"admin": {
					"class": "User",
					"userType": "regular",
					"shell": "bash"
				},
				"externalGwRoute": {
					"class": "Route",
					"target": "external",
					"network": "10.0.1.1/32",
					"mtu": 1460,
					"localOnly": false
				},
				"externalIntRoute": {
					"class": "Route",
					"gw": "10.0.1.1",
					"network": "10.0.1.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"default": {
					"class": "Route",
					"gw": "10.0.1.1",
					"network": "default",
					"mtu": 1460,
					"localOnly": false
				},
				"internalGwRoute": {
					"class": "Route",
					"target": "internal",
					"network": "10.0.2.1/32",
					"mtu": 1460,
					"localOnly": false
				},
				"internalIntRoute": {
					"class": "Route",
					"gw": "10.0.2.1",
					"network": "10.0.2.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"appRoute": {
					"class": "Route",
					"gw": "10.0.2.1",
					"network": "10.0.3.0/24",
					"mtu": 1460,
					"localOnly": false
				},
				"external": {
					"class": "VLAN",
					"tag": 4094,
					"mtu": 1460,
					"interfaces": [{
						"name": "1.0",
						"tagged": false
					}],
					"autoLastHop": "default",
					"cmpHash": "default",
					"failsafeEnabled": false,
					"failsafeAction": "failover-restart-tm",
					"failsafeTimeout": 90
				},
				"external-self": {
					"class": "SelfIp",
					"address": "10.0.1.11/32",
					"vlan": "external",
					"allowService": ["tcp:80", "tcp:443", "tcp:4353", "udp:1026"],
					"trafficGroup": "traffic-group-local-only"
				},
				"internal": {
					"class": "VLAN",
					"tag": 4093,
					"mtu": 1460,
					"interfaces": [{
						"name": "1.2",
						"tagged": false
					}],
					"autoLastHop": "default",
					"cmpHash": "default",
					"failsafeEnabled": false,
					"failsafeAction": "failover-restart-tm",
					"failsafeTimeout": 90
				},
				"internal-self": {
					"class": "SelfIp",
					"address": "10.0.2.11/32",
					"vlan": "internal",
					"allowService": ["udp:1026", "tcp:4353"],
					"trafficGroup": "traffic-group-local-only"
				},
				"configSync": {
					"class": "ConfigSync",
					"configsyncIp": "10.0.1.11"
				},
				"failoverAddress": {
					"class": "FailoverUnicast",
					"address": "10.0.1.11",
					"port": 1026
				},
				"failoverGroup": {
					"class": "DeviceGroup",
					"type": "sync-failover",
					"members": ["dd-ca02943f-bigip1.c.f5-7656-pdsoleng-dev.internal", "dd-ca02943f-bigip2.c.f5-7656-pdsoleng-dev.internal"],
					"owner": "/Common/failoverGroup/members/0",
					"autoSync": true,
					"saveOnAutoSync": false,
					"networkFailover": true,
					"fullLoadOnSync": false,
					"asmSync": false
				},
				"trust": {
					"class": "DeviceTrust",
					"localUsername": "admin",
					"remoteHost": "/Common/failoverGroup/members/0",
					"remoteUsername": "admin"
				}
			}
		}
	}
}
```

### Expected Behavior
Config should be applied (works in DO 1.28.0). Should roll back if failed.

### Actual Behavior
Both config and rollback fail.

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this issue to our internal product backlog as AUTOTOOL-3155.

**Comment by dstokesf5** [comment]
This has been resolved with the 1.31.0 release of Declarative Onboarding.
